scripts/views/eventView.js

```diff
@@ -9,10 +9,30 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
-    if (zip) {
-      TownHall.lookupZip($('#look-up input').val());
-      eventHandler.resetFilters();
+    if (zip && zip.length === 5) {
+      repZip = zip;
+    } else if (zip && zip.length > 5) {
+      zip = zip.match(/\d{5}/g)[0];
```

Comment:
If the user supplies more than 5 characters, but not 5 digits, this block proceeds and ends up throwing a console error (see my main comment on the PR when I tried "9abh1-eea4")